sup3r/models/base.py

```diff
@@ -717,12 +729,74 @@ def meta(self):
 
         return self._meta
 
+    @property
+    def s_enhance(self):
+        """Factor by which model will enhance spatial resolution. Used in
+        model training during high res coarsening"""
+        return self.meta.get('s_enhance', None)
+
+    @property
+    def t_enhance(self):
+        """Factor by which model will enhance temporal resolution. Used in
+        model training during high res coarsening"""
+        return self.meta.get('t_enhance', None)
+
+    @property
+    def smoothing(self):
+        """Value of smoothing parameter used in gaussian filtering of coarsened
+        high res data."""
+        return self.meta.get('smoothing', None)
+
+    def set_training_data_params(self, s_enhance, t_enhance, smoothing):
+        """Set parameters used for training data, such as s_enhance, t_enhance,
+        smoothing
+
+        Parameters
+        ----------
+        s_enhance : int
+            Spatial enhancement factor
+        t_enhance : int
+            Temporal enhancement factor
+        smoothing : float
+            Smoothing parameter used in gaussian filtering of coarsened data
+        """
+        if self.s_enhance is None:
+            self.meta['s_enhance'] = s_enhance
```

Comment:
You could make these three if/elif statements into a for loop. Maybe even have the input be kwargs so you could iterate easily.